doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -95,6 +95,20 @@ nRF Desktop
     See the documenation page of nRF Desktop's :ref:`nrf_desktop_hid_forward` for details.
   * Fixed an issue that was causing the HID keyboard LEDs to remain turned on after host disconnection while no other hosts were connected.
 
+Bluetooth® LE
+-------------
+
+* Updated:
+
+  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+
+Bluetooth mesh
+--------------
+
+* Updated:
+
+  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
```

Comment:
* Updated ~~some~~ samples with support for...

Reply:
I added `some` on @wiba-nordic  request :(

Reply:
Just a comment - not a blocker as I stated in previous comment.

Reply:
I'll ask the rest of the doc team for opinions.